app/helpers/key_pairs_helper.rb

```diff
@@ -0,0 +1,8 @@
+module KeyPairsHelper
+  def key_pair_status_icon(key_pair)
+    return icon_text('ok', '', {kind: 'pficon', class: 'center', title: _('Active key')}) if key_pair.active
+    return icon_text('ok', '', {kind: 'pficon', class: 'center warn',
+                                title: _('Key used with other compute resource')}) if key_pair.used_elsewhere
+    icon_text('circle-o', '', {kind: 'fa', class: 'center', title: _('Key not connected to compute resource')})
```

Comment:
this is any compute resource right? in that case might change the wording to any compute resource?